packages/eslint-plugin/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@angular-eslint/eslint-plugin",
-  "version": "19.0.2",
+  "version": "19.1.0",
```

Comment:
Revert this, versioning is automated and does not happen through imperative updates to version fields, and all packages are released together so changing only one package would never be the right thing to do.

Reply:
Reverted.